src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,27 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("Method completeOrder was called. Params: userId={}", userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
-        // NOTE: In production ready code this order identifier should be generated by DB
-        // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
         return order;
```

Comment:
The method 'completeOrder' always sets the orderId to a fixed value of 1. This would cause every order to have the same ID, which is not correct in a real-world application. Each order should have a unique identifier. You might want to implement an order ID generation strategy or use a database auto-increment feature.